Wireframe/css/style.css

```diff
@@ -0,0 +1,71 @@
+body {
+    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+    width: auto; 
+    margin-top: 40px;
+    background-color: beige;
+    margin-left: 120px;
+    margin-right: 100px;
+    margin-bottom: 50px;
+}
+
+.header {
```

Comment:
if you were to change the html to use semantic elements like `<header>` and so on you can remove the `.` and just refer to the element like `header`